data/shared/src/main/scala/sigma/utils/Overloading.scala

```diff
@@ -0,0 +1,15 @@
+package sigma.utils
+
+object Overloading {
```

Comment:
ScalaDocs missed

Reply:
This class turns out to be duplicated, removed.